priv/schema.json

```diff
@@ -81,10 +81,12 @@
         "introduction": {"$ref": "#/definitions/message"},
         "not_understood": {"$ref": "#/definitions/message"},
         "clarification": {"$ref": "#/definitions/message"},
+        "unsubscribe": {"$ref": "#/definitions/message"},
+        "unsubscribe_keyword": {"$ref": "#/definitions/message"},
```

Comment:
This should be a `localized_keywords` or `localized_string`, instead of a `message`. Keep in mind that the `message` object definition includes a `message` property inside, and then the `localized_string`. The difference between `localized_string` and `localized_keywords` is that the `localized_keywords` support an array of strings while the `localized_string` contains only one string per language. Maybe in this case we could assume that there will only be one unsubscribe keyword and use `localized_string`, but i'd rather use the `localized_keywords` as it doesn't add too much complexity and keeps the window open for allowing more than one keyword in the future.

Reply:
Even better, the whole idea of having a `messages` property in the `message` definition was for allowing for further expansion, say, having also a `keywords` property in there as well.

Support the following manifest structure:

```
"unsubscribe": {
  "message": {
    "en": "Send UNSUBSCRIBE to stop receiving messages",
    "es": "Enviá DESUSCRIBIR para dejar de recibir mensajes"
  },
  "keywords": {
    "en": ["unsubscribe"],
    "es": ["desuscribir"]
  },
  "acknowledge_message": {
    "en": "I won't send you any further messages",
    "es": "No te enviaré más mensajes"
  }
}
```